libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.ts

```diff
@@ -119,6 +119,13 @@ export class SkyFileAttachmentComponent
   @Input({ transform: booleanAttribute })
   public labelHidden = false;
 
+  /**
+   * [Persistent inline help text](https://developer.blackbaud.com/skyux/design/guidelines/user-assistance#inline-help) that provides
+   * additional context to the user.
+   */
```

Comment:
I believe we want the `@preview` tag on this?

Reply:
added